src/ui/components/NetworkMonitor/RequestRow.tsx

```diff
@@ -43,7 +43,7 @@ export const RequestRow = ({
             }}
           >
             <div
-              className={classNames("img", {
+              className={classNames("img shadow-md", {
```

Comment:
I think `shadow-md` needs to move up a level to show the shadow around the whole badge and not just the text.

Reply:
I think I'm confused here. Did I accidentally add a text-shadow somewhere instead of a box-shadow?

Reply:
No. this is a box shadow but it's on the icon and not the badge. My screenshot above shows it on the badge because i manually added it to verify but I didn't see the shadow in the original PR. Do you see the same?

Reply:
Ahhhhh, I see what you're saying. I thought I saw it but it might have just been an illusion of the lighter on darker color

Reply:
Yep, that's what it was. Thanks Ryan!